### PR TITLE
Refactor TWP23 lecture

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#33. 
**Note**: I didn't focus on redaction nor grammar. 
*TODO*: The final exercise is missing ActiveCode. I haven't added it yet because the `.txt` file the exercise uses has more than 3000 lines.  
[alice.txt](https://github.com/PyAr/PyZombis/files/6688286/alice.txt)

**UPDATE**: I added the final exercise already. I added the `alice.txt`  file to _static, and referenced it in a datafile directive in the lecture.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshot

https://user-images.githubusercontent.com/80920858/122974928-734de680-d358-11eb-8d9b-99004b0f6d47.mp4

### CODE REVIEW

File: _sources/lectures/TWP23.rst [hunk 1]
```diff
@@ -210,75 +321,88 @@ Diccionarios
...
-   :alt: 
+    :height: 12.571cm
+    :width: 9.411cm
+    :align: center
+    :alt: 
 
 
 + “La vida es como andar en bicicleta. Para mantener el equilibrio, debes seguir moviéndote ”. - Einstein
 
+.. datafile:: alice2.txt
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP23.rst:406]: This file shouldn't be on the last line. You cannot see the content.
image

Comment (LeoCumpli21) [reply _sources/lectures/TWP23.rst:406]: This is on purpose. That file has over 3000 lines. There's no need for the user to see it. That's why it is hidden with the parameter `:hide:`

Comment (angelasofiaremolinagutierrez) [reply _sources/lectures/TWP23.rst:406]: Oh, I understand. Maybe you can put the file before the exercise description where it's used (see image below). That way the student can see that the file exists. And I suggest to change the datafile name, maybe something like `alice_archivo.txt`

image

Comment (LeoCumpli21) [reply _sources/lectures/TWP23.rst]: Done 😄

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
LGTM :)

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**